tests/director/tests/formation_api_test.go

```diff
@@ -5010,11 +5344,68 @@ func getNotificationsFromExternalSvcMock(t *testing.T, client *http.Client) []by
 	return body
 }
 
-func assertFormationAssignmentsNotification(t *testing.T, notification gjson.Result, op, formationID, expectedAppID, expectedLocalTenantID, expectedAppNamespace, expectedAppRegion, expectedTenant, expectedCustomerID string) {
-	assertFormationAssignmentsNotificationWithConfig(t, notification, op, formationID, expectedAppID, expectedLocalTenantID, expectedAppNamespace, expectedAppRegion, expectedTenant, expectedCustomerID, nil)
+func assertNoDestinationIsFound(t *testing.T, client *clients.DestinationClient, destinationName string) {
+	_ = client.GetDestinationByName(t, destinationName, http.StatusNotFound)
```

Comment:
I explicitly wanted to have `_` because that way when you see the assert you know the method is returning something but we're not interested in it. But without `_` it seems the method is not returning anything(which is not the case).